setup.py

```diff
@@ -148,15 +148,15 @@ def run(self):
     packages=['azure.functions', 'azure.worker', 'azure.worker.protos'],
     provides=['azure.functions'],
     install_requires=[
-        'grpcio',
-        'grpcio-tools',
+        'grpcio~=1.9.1',
```

Comment:
can we move to latest version: 1.11.0?

Reply:
Can we use =

Reply:
`~` there pins the major.minor, but allows updating to the last patch version, which is generally better as that allows picking up non-breaking security fixes. Is absolute version pinning a requirement?

Reply:
Yea. Makes sense. pinning to a major version should be ok.